content/recipes/authoring/kata-snippets/description.md

```diff
@@ -5,6 +5,54 @@ prev: /recipes/authoring/kata-snippets/
+
+
+## General
+
+- **Description should hold to general [Content Quality Guidelines](/recipes/authoring/)** where applicable.
+- **Descriptions should be written in English.** The Codewars community is international and not every user can write English proficiently. Authors should ask other users for proofreading and spellchecking if necessary.
+- **All remarks and complaints about instructions not being clear should be considered with proper attention.** Authors should remember that after working on a kata for a long time, they understand it much better than others, and it's easy for them to miss some issues. Users may have completely different points of view, or simply understand things differently. Invite other users to provide feedback on description clarity, consider their suggestions, and integrate the feedback as appropriate.  
+When questions or requests for improvements, clarifications, or examples pop up particularly often, the description should be updated to address them.
+- **Descriptions should not make translating a kata into other programming languages difficult.** The description is shared between all translations of the kata and may need to be modified by translators. If there are many translations with many such modifications, description merge conflicts can occur, making kata maintenance difficult. As such, descriptions should ideally be as language-agnostic as possible and use only abstract terms not specific to any programming language. If a description cannot be kept language-agnostic, language-specific elements should be structured in a way which helps with resolution of merge conflicts: preferably in paragraphs or larger blocks rather than multiple language concepts in one sentence.
+
+
+## Structure
+
+- **Description should be structured into meaningful parts**, for example:
+  - overwiew, general context, or introduction,
+  - general statement of a task user has to solve,
+  - description of input and output, potentially with examples,
+  - detailed rules and requirements,
+  - additional notes, explanations, technical details, follow-ups.  
+Above is just an example; not all sections are necessary if not applicable, and authors can use another structure if they like. However, related pieces of information should be together: explaining inputs in 3-4 different places isn't a good idea.
+- **Descriptions should be consise, but precise.** Description of a task should be possibly short, because long descriptions suggest that task is either not very well defined, or is very complex. Fabularized or themed descriptions are OK as long as they don't get in a way of understanding the problem.
```

Comment:
```suggestion
- **Descriptions should be concise, but precise.** The description of a task should be as short as possible, because long descriptions suggest that the task is either not very well defined, or is very complex. Descriptions accompanied with a theme or storyline are acceptable as long as they don't get in the way of understanding the problem.
```

Reply:
It's a proper word in Polish, English could use it well :)